plugins/foreman_openscap/0.6/index.md

```diff
@@ -276,12 +280,18 @@ Go to 'SCAP Content' tab
 Select a Tailoring file from a dropdown and then a profile that comes with it
 ![Tailoring a Policy]({{page.images}}/policy_tailoring.png)
 
+### 4.5.4 Running a scan from UI ( foreman_openscap >= 0.6.5 )
+
+In version 0.6.5 and higher, you can initiate scans from UI. Simply go to hosts page and select "Run OpenSCAP scan" from the dropdown menu. This will initiate a scan for all policies that are assigned to the host. foreman_remote_execution plugin needs to be installed for this feature to be enabled.
```

Comment:
> foreman_remote_execution plugin needs to be installed

maybe mentioning the version would be useful, it's currently in master only but next rex version should be 1.2.3, I kindly ask @adamruzicka to confirm the next version number